drivers/sensor/apds9306/apds9306.c

```diff
@@ -230,6 +232,7 @@ static int apds9306_attr_get(const struct device *dev, enum sensor_channel chann
 			     enum sensor_attribute attribute, struct sensor_value *value)
 {
 	struct apds9306_data *data = dev->data;
+	value->val2 = 0;
```

Comment:
Perhaps better to do this after the error handling so you don't end up setting val2 only and then return an error.